api/app/tests/fba/test_fba_error_random_sample.py

```diff
@@ -0,0 +1,239 @@
+                                                   time_of_interest=time_of_interest,
+                                                   percentage_conifer=percentage_conifer,
+                                                   percentage_dead_balsam_fir=percentage_dead_balsam_fir,
+                                                   grass_cure=grass_cure,
+                                                   crown_base_height=crown_base_height,
+                                                   lat=latitude,
+                                                   long=longitude,
+                                                   bui=bui,
+                                                   ffmc=ffmc,
+                                                   isi=isi,
+                                                   wind_speed=wind_speed,
+                                                   wind_direction=wind_direction,
+                                                   temperature=temperature,  # temporary fix so tests don't break
+                                                   relative_humidity=relative_humidity,
+                                                   precipitation=precipitation,
+                                                   status='Forecasted',
+                                                   prev_day_daily_ffmc=None,
+                                                   last_observed_morning_rh_values=None)
+        python_fba = calculate_fire_behaviour_advisory(python_input)
+        # get REDapp result from java:
+        java_fbp = FBPCalculateStatisticsCOM(elevation=elevation,
+                                             latitude=latitude,
+                                             longitude=longitude,
+                                             time_of_interest=get_hour_20_from_date(time_of_interest),
+                                             fuel_type=fuel_type,
+                                             ffmc=ffmc,
+                                             dmc=dmc,
+                                             dc=dc,
+                                             bui=bui,
+                                             wind_speed=wind_speed,
+                                             wind_direction=wind_direction,
+                                             percentage_conifer=percentage_conifer,
+                                             percentage_dead_balsam_fir=percentage_dead_balsam_fir,
+                                             grass_cure=grass_cure,
+                                             crown_base_height=crown_base_height)
+        results.append({
+            'input': {'isi': isi, 'bui': bui, 'wind_speed': wind_speed, 'ffmc': ffmc},
+            'python': python_fba,
+            'java': java_fbp,
+            'fuel_type': fuel_type}
+        )
+
+    return results
+
+
+@ then("ROS is within <ros_margin_of_error> compared to REDapp")
```

Comment:
file has `@given` but `@ then` - would prefer to see consistency in formatting. (My vote is for no space)

Reply:
ggrrr. stupid autoformat.